backend/lib/adapters/gateways/acms/acms.gateway.test.ts

```diff
@@ -40,21 +39,43 @@ describe('ACMS gateway tests', () => {
     const gateway = new AcmsGatewayImpl(context);
     await gateway.getLeadCaseIds(context, predicate);
 
-    if (params.inputVariable) {
-      expect(spy).toHaveBeenCalledWith(
-        context,
-        expect.any(String),
-        expect.arrayContaining([
-          expect.objectContaining({ name: 'chapter', value: params.inputVariable }),
-        ]),
-      );
-    } else {
-      expect(spy).toHaveBeenCalledWith(
-        context,
-        expect.stringContaining(params.querySubString),
-        expect.any(Array),
-      );
-    }
+    expect(spy).toHaveBeenCalledWith(
+      context,
+      expect.any(String),
+      expect.arrayContaining([
+        expect.objectContaining({ name: 'chapter', value: params.inputVariable }),
+      ]),
+    );
+  });
+
+  test('should handle chapter 7 query', async () => {
+    const spy = jest
+      .spyOn(AbstractMssqlClient.prototype, 'executeQuery')
+      .mockResolvedValueOnce({
+        success: true,
+        results: [{ leadCaseCount: 0 }],
+        message: '',
+      })
+      .mockResolvedValue({
+        success: true,
+        results: [],
+        message: '',
+      });
+
+    const predicate: AcmsPredicate = {
+      chapter: '7',
+      divisionCode: '081',
+    };
+
+    const context = await createMockApplicationContext();
+    const gateway = new AcmsGatewayImpl(context);
+    await gateway.getLeadCaseIds(context, predicate);
+
+    expect(spy).toHaveBeenCalledWith(
+      context,
+      expect.stringContaining("IN ('7A', '7N')"),
+      expect.any(Array),
+    );
```

Comment:
This probably answers my question above ;-)

Reply:
Indeed it does.